### PR TITLE
Implementation of function getItems(String..) in class

### PR DESCRIPTION
**Pull request by jkopriva**
org.eclipse.reddeer.swt.impl.table.AbstractTable

Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
Looks like #1962 contains fix. After merging this one, this one can be verified by build as well.

File: plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/api/Table.java [hunk 1]
```diff
@@ -145,6 +145,16 @@
 	@SuppressWarnings("unchecked")
 	List<TableItem> getItems(Matcher<TableItem>... matchers);
 	
+	/**
+	 * Returns table items matching the strings in columns(fixed order).
+	 * Returns 0, if array of string is longer than number of columns.
```
Comment (odockal) [inline plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/api/Table.java]: Returns an empty array sounds better here, right?

**Comment by jkopriva** [comment]
@odockal thx for review and merge :)

**Pull request merged**